galsim/utilities.py

```diff
@@ -1148,6 +1148,9 @@ def combine_wave_list(*args):
     wave_list = wave_list[(wave_list >= blue_limit) & (wave_list <= red_limit)]
     if blue_limit > red_limit:
         raise RuntimeError("Empty wave_list intersection.")
+    # Make sure both limits are included.
```

Comment:
I left in the `has_attr`, since I don't think it makes much sense to give red/blue limits to ChromaticObjects.  But good point about the None's no longer being possible.